src/abstractitensornetwork.jl

```diff
@@ -150,6 +189,14 @@ function siteinds(tn::AbstractITensorNetwork, vertex...)
   return uniqueinds(tn, vertex...)
 end
 
+function siteinds(::typeof(all), tn::AbstractITensorNetwork, vertex...)
+  return siteinds(tn, vertex...)
+end
+
+function siteinds(::typeof(only), tn::AbstractITensorNetwork, vertex...)
```

Comment:
I eventually regretted adding functions like this. I think it is easy enough to write `only(siteinds(tn, v))`.

Reply:
Also, I've decide in the latest versions of `NamedGraphs.jl` and `DataGraphs.jl` to not allow splatted vertex indexing in general (except in the case of `data_graph[1, 1]` for getting vertex data), so this and related functions should just be:
```julia
function siteinds(tn::AbstractITensorNetwork, vertex)
  [...]
end
```

Reply:
I removed the extra filters for `siteinds` and `linkinds`, as well as all splatted vertex indexing. I don't even remember why I added the filters in the first place, I must have been overloading some other `ITensors.jl` function that used this interface and added them without thinking.

Reply:
No problem. There is going to be some tension between the MPO/MPS types in ITensors.jl and AbstractITensorNetwork types in ITensorNetworks.jl in terms of the interface, since I think ITensorNetworks.jl is an opportunity to rethink some of the MPO/MPS interface using a more general network perspective.